cmd/peers.go

```diff
@@ -26,17 +26,17 @@ var peersInfoCmd = &cobra.Command{
 	PreRun: func(cmd *cobra.Command, args []string) {
 		loadConfig()
 	},
+	Args: cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) < 1 {
-			ErrExit(fmt.Errorf("peer name is required"))
-		}
+		req, err := peerRequests(false)
```

Comment:
Just to double-check, this is changing to work for offline peers?

Reply:
kinda, this `false` value is a bit of a misnomer, I'm planning on removing it entirely in the future, it currently only sets if a command _must_ be connected to the p2p network to work. But generally speaking this value is stupid, and should be ignored until we kill it.